### DISCUSSION TITLE
How to resume project?

### DISCUSSION THREAD

**Discussion by nicitaacom** [supabase/supabase#20377] [Questions] [answered]
![I got this](https://i.imgur.com/nh7iJ5J.png)

I got this when I want to resume my paused project

I have 2 projects in this organization - as I see system think that I'm going to create third project - but I just want to unpause current

**Comment by nicitaacom** [comment]
Also what?
Supbase message me that I may have only 2 projects in organization but I can create 3 projects in 1 organization

![what](https://i.imgur.com/W1qyOoP.png)

**Comment by nicitaacom** [comment]
Also what the UX?

![image](https://github.com/supabase/supabase/assets/39565703/036bda65-050e-4bb4-a967-9647b09299e0)


I know that this project already paused - I want to unpase it

**Reply by pcDamasceno** [reply]
How did you unpause it?

**Reply by nicitaacom** [reply]
@pcDamasceno I don't remember

**Comment by awalias** [comment, marked as answer]
Hey @nicitaacom can you please open a support ticket for this https://github.com/orgs/supabase/discussions/20377

**Reply by nicitaacom** [reply]
I don't know why but I got notification jsut today
I don't want to do that
I think I already solved problem
